fbchat/client.py

```diff
@@ -1485,29 +1572,48 @@ def changeThreadEmoji(self, emoji, thread_id=None):
 
         j = self._post(self.req_url.THREAD_EMOJI, data, fix_request=True, as_json=True)
 
-    def reactToMessage(self, message_id, reaction):
-        """
-        Reacts to a message
-
-        :param message_id: :ref:`Message ID <intro_message_ids>` to react to
-        :param reaction: Reaction emoji to use
-        :type reaction: models.MessageReaction
-        :raises: FBchatException if request failed
-        """
-        full_data = {
+    def _react(self, message_id, reaction=None):
+        data = {
             "doc_id": 1491398900900362,
             "variables": json.dumps({
                 "data": {
-                    "action": "ADD_REACTION",
+                    "action": "ADD_REACTION" if reaction else "REMOVE_REACTION",
                     "client_mutation_id": "1",
                     "actor_id": self.uid,
                     "message_id": str(message_id),
-                    "reaction": reaction.value
+                    "reaction": reaction.value if reaction else None
                 }
             })
         }
 
-        j = self._post(self.req_url.MESSAGE_REACTION, full_data, fix_request=True, as_json=True)
+        r = self._post(self.req_url.MESSAGE_REACTION, data)
+        r.raise_for_status()
```

Comment:
Use `fix_request` in `Client._post` instead of `r.raise_for_status()`. And when the response is actually json, use `as_json` as well, that allows for better error handling